src/components/bannerCard.tsx

```diff
@@ -0,0 +1,246 @@
+                            <Button
+                                iconProps={{ iconName: "Edit" }}
+                                text={this.state.expanded === true ? "Edit less" : "Edit more"}
+                                onClick={() => {
+                                    this.setState({ expanded: !this.state.expanded });
+                                }}
+                            />
+                            <Button
+                                iconProps={{ iconName: "Save" }}
+                                primary={true}
+                                disabled={!this.isSaveEnabled()}
+                                onClick={() => this.saveBanner()}
+                                tooltipProps={{ text: "Save" }}
+                            />
+                            <Button
+                                iconProps={{ iconName: "Delete" }}
+                                danger={true}
+                                onClick={() => this.deleteBanner()}
+                                tooltipProps={{ text: "Delete" }}
+                            />
+                        </ButtonGroup>
+                    </div>
+                    {this.state.expanded === true ? <div className="banner-body">
+                        <FormItem label="Level" className="level-dropdown">
+                            <LevelDropdown
+                                level={this.state.level}
+                                onChange={(level) => {
+                                    this.setState({ level });
+                                    this.ensureMarkedDirty();
+                                }}
+                            />
+                        </FormItem>
+                        <ExpiryPicker
+                            expiryDate={this.state.expirationDate}
+                            onChange={(expirationDate) => {
+                                this.setState({ expirationDate, expiryErrorText: null });
+                                this.ensureMarkedDirty();
+                            }}
+                            onParseError={(errorMessage) => {
+                                this.setState({ expiryErrorText: errorMessage });
+                            }}
+                        />
+                    </div> : null}
+                </div>
+            </Card>
+        );
+    }
+
+    private getMessageError(message: string): string {
+        if (message.split(" ").length >= 30) {
+            return "Message too long";
+        }
+
+        return null;
+    }
+
+    private async saveBanner(): Promise<void> {
+        this.setState({ loading: true });
+
+        const banner = this.props.banner;
+        banner.message = this.state.message;
+        banner.level = this.state.level;
+        banner.expirationDate = this.state.expirationDate;
+
+        try {
+            await VstsService.instance.saveWebGlobalMessageBanner(banner);

```

Comment:
Can we drop using ".instance" somehow ?

Reply:
I am using export default. Let me know what you think.